shared/src/commonMain/kotlin/io/customer/shared/tracking/model/Activity.kt

```diff
@@ -1,10 +1,13 @@
+@file:UseContextualSerialization(Any::class)
```

Comment:
Another place that could use a comment explanation.

Reply:
Sure, will do.